api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
Also note the known issue that the gateway does not forward `Host` headers through today (near the bottom of http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-known-issues.html). So we'd have to hard-code the gateway's custom domain to a stage variable, we can't dynamically figure it out by forwarding the host header from the original request.